### ISSUE TITLE
Release?

### ISSUE THREAD

**Issue by larsoner** [enthought/mayavi#1294]
@prabhuramachandran now that NumPy [2.0.0rc1 is out](https://pypi.org/project/numpy/2.0.0rc1/) I think it's a good time to cut a release. Ideally it needs to be done in the next couple of weeks otherwise people's installs of Mayavi will start breaking I think.

**Comment by prabhuramachandran** [comment]
Good idea, are there any PRs that need to be looked at or is this just a matter of going through the release process?

**Comment by larsoner** [comment]
ping @prabhuramachandran

**Comment by prabhuramachandran** [comment]
@larsoner -- apologies, been a busy semester. Just wrapped up a few days ago. Thanks for the reminder. Will try to get this done in the next few days.

**Comment by prabhuramachandran** [comment]
@larsoner -- I've pushed one change before the release in #1300. Once that is merged, I will work on the release. I found these issues when testing the latest version on Python 3.12 using pip to ensure that it all works. I've tested this on PyQt5 and 6 and both seem to work fine. There seem to be test errors which I will look into.
